spec/yaml/swiftnav/sbp/ndb.yaml

```diff
@@ -0,0 +1,80 @@
+ # Copyright (C) 2017 Swift Navigation Inc.
+# Contact: Pasi Miettinen <pasi.miettinen@exafore.com>
+#
+# This source is subject to the license found in the file 'LICENSE' which must
+# be be distributed together with this source. All other rights reserved.
+#
+# THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+# EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+
+package: swiftnav.sbp.ndb
+description: |
+  Messages for logging NDB events.
+stable: False
+public: False
+include:
+  - types.yaml
+  - gnss.yaml
+definitions:
+ - MSG_NDB_EVENT:
+    id: 0x0400
+    public: false
+    short_desc: Navigation DataBase Event
+    desc: |
+      This message is sent out when an object is stored into NDB. If needed
+      message could also be sent out when fetching an object from NDB.
+    fields:
+      - recv_time:
+          type: u64
+          units: ms
+          desc: HW time in milliseconds.
+      - event:
+          type: u8
+          desc: |
```

Comment:
can you use the `fields` tag to describe this and all other flag/enum type fields?